<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
    <arg name="config" default="baseline" doc="Specify a config."/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />

    <arg name="im_dir" default="" doc="Directory path of test images."/>
    <arg name="num_im" default="0" doc="Number of test images."/>
    
    <arg name="pkg_name" value="mdoap"/>
    <arg name="node_name" default="static_object_detector_tester_node"/>

    <group ns="$(arg veh)">
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
              <param name="~im_dir" value="$(arg im_dir)"/>
              <param name="~num_im" value="$(arg num_im)"/>
         </node>
    </group>
    
    <include file="$(find mdoap)/launch/static_object_detector_node.launch">
         <arg name="veh" value="$(arg veh)"/>
         <arg name="local" value="true" />
         <arg name="config" value="$(arg config)"/>
         <arg name="param_file_name" value="$(arg param_file_name)"/>
    </include>
</launch>